### PR TITLE
 #162754321 Test models

### PR DESCRIPTION
**Pull request by Georgeygigz**
# What does this PR do?
It creates test for to do list model

# Description of Task to be completed?
Have tests for to do list model

# How should this be manually tested?
Clone the project locally

cd into the Bucketlist folder

# What are the relevant pivotal tracker stories?
162754321

### CODE REVIEW

File: djangorestapi/api/tests.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
 from django.test import TestCase
```
Comment (KMaina) [inline djangorestapi/api/tests.py]: Add a doc string at the top of the file

Comment (Georgeygigz) [reply djangorestapi/api/tests.py]: Okay will check on that.

Comment (andrewhingah) [reply djangorestapi/api/tests.py]: Use the Andela best practices for naming the PRs

File: djangorestapi/api/tests.py [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+from .models import ToDoList
 
-# Create your tests here.
```
Comment (KMaina) [inline djangorestapi/api/tests.py]: Replace this with a doc string

File: djangorestapi/api/tests.py (hunk 1 above)
Comment (KMaina) [inline djangorestapi/api/tests.py:3]: Add a doc string here at the top of the file

**Pull request merged**